backend/src/Altinn.Authorizationadmin/Altinn.Authorizationadmin/Migration/v0.07-authorization-admin/02-setup-procedures.sql

```diff
@@ -1,22 +1,22 @@
 --Procedure: delegation.InsertResourceDelegation
-CREATE OR REPLACE FUNCTION delegation.InsertResourceDelegation(
-  _delegationChangeType delegation.delegationChangeType,
-  _altinnappid character varying,
-  _offeredbypartyid integer,
-  _coveredbyuserid integer,
-  _coveredbypartyid integer,
-  _performedbyuserid integer,
-  _blobstoragepolicypath character varying,
-  _blobstorageversionid character varying,
-  _resourceid character varying,
-  _resourcetype character varying
-)
-RETURNS SETOF delegation.delegationchanges 
-LANGUAGE 'sql'
-VOLATILE
-ROWS 1
+CREATE OR REPLACE FUNCTION delegation.insert_delegationchange(
+	_delegationchangetype delegation.delegationchangetype,
+	_altinnappid character varying,
+	_offeredbypartyid integer,
+	_coveredbyuserid integer,
+	_coveredbypartyid integer,
+	_performedbyuserid integer,
+	_blobstoragepolicypath character varying,
+	_blobstorageversionid character varying,
+	_resourceid character varying,
+	_resourcetype character varying)
+    RETURNS SETOF delegation.delegationchanges 
+    LANGUAGE 'sql'
+    VOLATILE
+    ROWS 1
+
 AS $BODY$
-  INSERT INTO delegation.delegationChanges(
+INSERT INTO delegation.delegationChanges(
```

Comment:
Missing 01-setup-tables.sql script for actually adding the columns to the delegationChanges table?

Reply:
No changes in 01-setup-tables.sql in this PR but the script exists in repo.